chrome/manifest.json

```diff
@@ -42,7 +42,8 @@
   "offline_enabled": false,
   "permissions": [
     "tabs", 
```

Comment:
I am not a chrome plugin developer but I am interested in this addon. Is this permission really still needed? Does it have something to do with openend tabs? When I try to install the extension from the store, it stilly says that it can read and change all data on all websites that I visit.